qutebrowser/app.py

```diff
@@ -806,6 +806,18 @@ def __init__(self, args):
         self.launch_time = datetime.datetime.now()
         self.focusObjectChanged.connect(self.on_focus_object_changed)
 
+    def event(self, e):
+        if e.type() != QEvent.FileOpen:
+            return super(QApplication, self).event(e)
```

Comment:
You can just do `super()` here as we use Python 3 only

Reply:
done